src/Cli/dotnet/Extensions/ParseResultExtensions.cs

```diff
@@ -97,13 +97,13 @@ public static bool IsDotnetBuiltInCommand(this ParseResult parseResult)
 
     public static bool IsTopLevelDotnetCommand(this ParseResult parseResult)
     {
-        return parseResult.CommandResult.Command.Equals(RootCommand) && string.IsNullOrEmpty(parseResult.RootSubCommandResult());
+        return parseResult.CommandResult.Command.Equals(Microsoft.DotNet.Cli.Parser.RootCommand) && string.IsNullOrEmpty(parseResult.RootSubCommandResult());
```

Comment:
`CliRootCommand` got renamed to `RootCommand` and it conflicted with `RootCommand` which was imported here with a static using:

https://github.com/dotnet/sdk/blob/afb349499d82e3606cf42ab0bcdc00b3d33557f5/src/Cli/dotnet/Extensions/ParseResultExtensions.cs#L11